docs/file_format.rst

```diff
@@ -38,6 +38,16 @@ The optional ``extracellular`` key specifies the default string for
 the extracellular compartment on compounds. If this option is not
 specified it will be assumed that the extracellular compartment is called ``e``.
 
+-Default Compartment
+--------------------------
+
+The optional ``default compartment`` key specifies the default compartment
+that is used if a compound in a reaction does not explicitly specify a
+compartment.  For example, the reaction |x[e]| + |atp| => |x| + |adp| + |pi|
```

Comment:
The reaction should be ``` `` ```-quoted so it is displayed as fixed-width font. I don't think the `default compartment` is correct, there should be an underscore in there. Did you check with the code that it actually parses `default_compartment` and not `default`?